addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -3097,6 +3103,16 @@ options.registry.WebsiteAnimate = options.Class.extend({
             this.$optionsSection.append(this.$el);
         }
     },
+    /**
+     * @override
+     */
+    cleanForSave() {
+        if (this.$target[0].closest('.o_animate')) {
+            // As images may have been added in an animated element, we must
+            // remove the lazy loading on them.
+            this._toggleImagesLazyLoading(false);
+        }
+    },
```

Comment:
we need it in the `cleanForSave` for image that we add in already animated element (as explained in the comment)

Reply:
My bad, misunderstood the comment :shrug: Although, it could be better to listen to `content_changed` in this animate snippet option to detect when an image is added. But maybe too overkill, it's fine as it is. I just worry each time I see a `cleanForSave` because I know the code linked to it is not robust enough